plutus-core/executables/Common.hs

```diff
@@ -230,7 +230,8 @@ printBudgetStateTally term model (Cek.CekExTally costs) = do
           (getCPU $ getSpent Cek.BStartup) + getCPU totalComputeCost + getCPU builtinCosts
 
 class PrintBudgetState cost where
-    printBudgetState :: UPLC.Term PLC.Name PLC.DefaultUni PLC.DefaultFun () -> CekModel -> cost -> IO ()
+    printBudgetState :: UPLC.Term PLC.Name PLC.DefaultUni PLC.DefaultFun ()
```

Comment:
Usually when we break things over multiple lines we go "all the way", so this would become something like:
```
printBudgetState 
  :: UPLC.Term ..
  -> CekModel
  -> cost
  -> IO ()
```
just prevents you from having to think about where to break it.